server/channels/app/users/helper_test.go

```diff
@@ -69,7 +70,9 @@ func setupTestHelper(s store.Store, includeCacheLayer bool, tb testing.TB) *Test
 	*config.PasswordSettings.Uppercase = false
 	*config.PasswordSettings.Symbol = false
 	*config.PasswordSettings.Number = false
-	configStore.Set(config)
+
+	_, _, err = configStore.Set(config)
+	require.NoError(tb, err, "Failed to set config")
```

Comment:
Just `require.NoError(tb, err)` is sufficient.

Reply:
I've fixed the issue.